src/seedu/addressbook/commands/ViewCommand.java

```diff
@@ -5,14 +5,14 @@
 
 /**
  * Shows details of the person identified using the last displayed index.
- * Private contact details are not shown.
+ * Private contact details, exams are not shown.
  */
 public class ViewCommand extends Command {
 
     public static final String COMMAND_WORD = "view";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ":\n" + "Shows the non-private details of the person "
-            + "identified by the index number in the last shown person listing.\n\t"
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ":\n" + "Shows the non-private details excluding exams "
+            + "of the person identified by the index number in the last person listing.\n\t"
             + "Parameters: INDEX\n\t"
```

Comment:
The wording of this command usage I think needs to be discussed. Cause if we follow you, then we'll have a long chain of excluding A,B,C,D,E,F. So I think it's something we need to decide as a group as to how we want to describe this command

Reply:
theres many things to discuss as a group...